pkg/scheduler/plugins/deviceshare/utils.go

```diff
@@ -117,7 +117,7 @@ var ValidateGPURequest = func(podRequest corev1.ResourceList) (uint, error) {
 		return gpuCombination, nil
 	}
 
-	return gpuCombination, fmt.Errorf("request is not valid, current combination: %b", gpuCombination)
+	return gpuCombination, fmt.Errorf("request is not valid, current combination: %v", convertGPURequestCombination(gpuCombination))
```

Comment:
maybe we can give what is right?

Reply:
![image](https://user-images.githubusercontent.com/24452340/223123251-435bccac-9c0b-4a54-b405-39f690d64813.png)

Reply:
hmm.. we should update the comments..